core/controllers/base.py

```diff
@@ -413,7 +413,7 @@ def validate_and_normalize_args(self):
         # This check ensures that if a request method is not defined
         # in the handler class then schema validation will not raise
         # NotImplementedError for that corresponding request method.
-        if request_method in ['GET', 'POST', 'PUT', 'DELETE'] and (
+        if request_method in ['GET', 'POST', 'PUT', 'DELETE', 'HEAD'] and (
```

Comment:
I think that for the HEAD we should use the same validation as for GET

Reply:
done

Reply:
Are you sure I would expect to just return the headers for the GET if HEAD comes.